app/views/admin/configs/show.html.erb

```diff
@@ -175,30 +178,33 @@
                     <%= inline_svg_tag("email.svg", class: "crayons-icon", aria: true, title: "Twitter logo") %>
                   </figure>
                   <div class="config-authentication__item--container">
-                    <div class="config-authentication__item--label">
-                      <p class="fw-medium">
-                        Email address and password
-                      </p>
+                    <div style="display: flex; justify-content: space-between; align-items: baseline;">
```

Comment:
@lisasy I need help locating a class that covers the CSS properties in this `<style>` tag here 🙏